### PR TITLE
DocuSign implementation

### PR DESCRIPTION
**Pull request by caleeli**
This PR includes:
- Ability to register PMFunctions from packages:
```
$manager = app('workflow.FormalExpression');
$manager->registerPMFunction('foo', function ($data, $param1) {
    return 'bar';
);
```
- Enable interstitial ability for CallActivity

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

**Review by boliviacoca — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

File: ProcessMaker/Models/FormalExpression.php [hunk 1]
```diff
@@ -187,9 +207,19 @@ function ($__data,$str) {
             function () {
             },
             function ($arguments, $array, $key, $default) {
-                return Arr::get($array,$key, $default);
+                return Arr::get($array, $key, $default);
             }
         );
+
+        // Register global PM functions
```
Comment (danloa) [inline ProcessMaker/Models/FormalExpression.php:214]: Specify that this code is used to register functions from packages

File: ProcessMaker/Repositories/TokenRepository.php [hunk 2]
```diff
@@ -100,7 +100,9 @@ public function persistActivityActivated(ActivityInterface $activity, TokenInter
         $token->updateTokenProperties();
         $token->saveOrFail();
         $token->setId($token->getKey());
-        event(new ProcessUpdated($token->getInstance(), 'ACTIVITY_ACTIVATED'));
+        /** @var ProcessRequest $request */
```
Comment (danloa) [inline ProcessMaker/Repositories/TokenRepository.php:103]: Remove this non standard annotations

**Review by danloa — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

**Review by danloa — APPROVED** [review]

**Review by boliviacoca — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://develop-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-3104b-processmaker.ci.processmaker.net

**Pull request merged**